python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -247,6 +247,10 @@ async def _tokenize_one_request(
         # Parse sampling parameters
         sampling_params = SamplingParams(**obj.sampling_params)
         sampling_params.normalize(self.tokenizer)
+        sampling_params.update_from_config(
```

Comment:
Revert the changes in `tokenizer_manager.py` and `sampling_params.py`. Move all related changes to `schedule_batch.py`.
Reason:
The `sampling_params` is used to handle per-request configs, but this eos_token_ids form hf_config is the same for all requests, so we do not need to attach it to `sampling_params`.
To reduce the overhead, we should pre-process the hf-config and add some more conditions in `check_finished`

Reply:
Add an input argument for `check_finished` as Req cannot acquire model info.